core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -3001,6 +3002,61 @@ class WorkspaceServiceSpec
     response.workspace.state shouldBe WorkspaceState.Ready
   }
 
+  it should "get the details of an Azure workspace that is deleting" in withTestDataServices { services =>
+    val service = services.workspaceService
+    val workspaceId1 = UUID.randomUUID().toString
+
+    val deletingAzureWorkspace =
+      Workspace.buildMcWorkspace("test_namespace1",
+                                 "name",
+                                 workspaceId1,
+                                 new DateTime(),
+                                 new DateTime(),
+                                 "testUser1",
+                                 Map.empty,
+                                 WorkspaceState.Deleting
+      )
+
+    runAndWait {
+      for {
+        _ <- slickDataSource.dataAccess.workspaceQuery.createOrUpdate(deletingAzureWorkspace)
+      } yield ()
+    }
+
+    when(service.workspaceManagerDAO.getWorkspace(deletingAzureWorkspace.workspaceIdAsUUID, services.ctx1))
+      .thenReturn(
+        new WorkspaceDescription().stage(WorkspaceStageModel.MC_WORKSPACE)
+      ) // no azureContext, should not be returned
```

Comment:
It was a comment that was already there… it's trying to point out that an azureContext is explicitly NOT being returned. If it were returned, it would look like this:

```
      when(service.workspaceManagerDAO.getWorkspace(azureWorkspace.workspaceIdAsUUID, services.ctx1))
        .thenReturn(
          new WorkspaceDescription()
            .stage(WorkspaceStageModel.MC_WORKSPACE)
            .azureContext(
              new AzureContext()
                .tenantId(UUID.randomUUID.toString)
                .subscriptionId(UUID.randomUUID.toString)
                .resourceGroupId(UUID.randomUUID.toString)
            )
        )